bouncerscript/tests/test_script.py

```diff
@@ -153,13 +158,26 @@ async def test_bouncer_aliases(aliases_context, mocker):
                 {"os": "win", "id": "47767", "path": "/firefox/nightly/latest-mozilla-central/firefox-63.0a1.en-US.win32.installer.exe"},
                 {"os": "linux", "id": "47764", "path": "/firefox/nightly/latest-mozilla-central/firefox-63.0a1.en-US.win64.installer.exe"},
             ],
+            True,
+            False,
+        ),
+        (
+            [
+                {"os": "win", "id": "47767", "path": "/firefox/nightly/latest-try/firefox-62.0a1.en-US.win32.installer.exe"},
+                {"os": "linux", "id": "47764", "path": "/firefox/nightly/latest-try/firefox-62.0a1.en-US.win64.installer.exe"},
+            ],
+            [
+                {"os": "win", "id": "47767", "path": "/firefox/nightly/latest-try/firefox-63.0a1.en-US.win32.installer.exe"},
+                {"os": "linux", "id": "47764", "path": "/firefox/nightly/latest-try/firefox-63.0a1.en-US.win64.installer.exe"},
```

Comment:
If I'm reading this right we're testing with successive versions here, but we should probably have a test that non-successive versions don't raise?

Reply:
Right yes, thank you!